dpnp/backend/src/dpnp_fptr.hpp

```diff
@@ -219,18 +219,6 @@ template <typename _Tp>
 using dpnp_remove_cvref_t =
     typename std::remove_cv_t<typename std::remove_reference_t<_Tp>>;
 
-/**
```

Comment:
`get_id_by_xyz_inkernel` and `get_result_shape` can be removed in `dpnp/backend/src/dpnp_utils.hpp`